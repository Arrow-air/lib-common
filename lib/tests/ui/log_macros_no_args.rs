use lib_common::log_macros;

log_macros!();

fn main() {}
