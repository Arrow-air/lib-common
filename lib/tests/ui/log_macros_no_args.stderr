error: log_macros takes at least 1 argument
 --> tests/ui/log_macros_no_args.rs:3:1
  |
3 | log_macros!();
  | ^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `log_macros` (in Nightly builds, run with -Z macro-backtrace for more info)
